sequencer/status.go

```diff
@@ -0,0 +1,47 @@
+package sequencer
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/dymensionxyz/roller/cmd/consts"
+	"github.com/dymensionxyz/roller/config"
+	"io/ioutil"
+	"net/http"
+)
+
+type NodeInfo struct {
+	Network string `json:"network"`
+}
+
+type Result struct {
+	NodeInfo NodeInfo `json:"node_info"`
+}
+
+type Response struct {
+	Result Result `json:"result"`
+}
+
+func IsSeqListen(rollappID string) bool {
+	resp, err := http.Get(fmt.Sprintf("%s/status", consts.DefaultRollappRPC))
+	if err != nil {
+		return false
+	}
+	defer resp.Body.Close()
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return false
+	}
+	var response Response
+	if err := json.Unmarshal(body, &response); err != nil {
+		return false
+	}
+	return response.Result.NodeInfo.Network == rollappID
```

Comment:
what u checking here? why not just check for 200OK?

Reply:
Because it can be another network. As discussed on #197, sometimes the sequencer does not terminates, which can cause the endpoint to return a healthy status when it is running another rollapp.